src/ucp/tag/offload.c

```diff
@@ -78,7 +92,8 @@ void ucp_tag_offload_completed(uct_tag_context_t *self, uct_tag_t stag,
                                UCP_RECV_DESC_FLAG_LAST);
         ucs_mpool_put_inline(req->recv.rdesc);
     } else {
-        iface = ucp_tag_offload_iface(worker);
+        iface = ucp_tag_offload_iface(worker, stag);
```

Comment:
looks like merge conflict, ucp_request_recv_buffer_dereg(req) should be used here